build.psm1

```diff
@@ -597,16 +597,27 @@ Fix steps:
         Pop-Location
     }
 
-    if ($Environment.IsRedHatFamily) {
+    if ($Environment.IsRedHatFamily -or $Environment.IsDebian9) {
         # add two symbolic links to system shared libraries that libmi.so is dependent on to handle
         # platform specific changes. This is the only set of platforms needed for this currently
         # as Ubuntu has these specific library files in the platform and macOS builds for itself
         # against the correct versions.
+
+        if ($Environment.IsDebian9){
+            # NOTE: Debian 8 doesn't need these symlinks
+            $sslTarget = "/usr/lib/x86_64-linux-gnu/libssl.so.1.0.2"
+            $cryptoTarget = "/usr/lib/x86_64-linux-gnu/libcrypto.so.1.0.2"
```

Comment:
I wonder that it doesn't work in CoreFX https://github.com/janvorli/corefx/blob/13e2cc977713f0a5edc7a702583eeabb5643006d/src/Native/Unix/System.Security.Cryptography.Native/opensslshim.cpp#L43 (from https://github.com/dotnet/corefx/pull/27208)

Reply:
I suspect it's because they are loading explicitly by version; libmi doesn't have version detection logic.

Reply:
Do you mean that a root problem in libmi? if so it seems better to make a correction there than to receive surprises with each new linux distributive.